l10n_it_fatturapa_in/__manifest__.py

```diff
@@ -1,7 +1,7 @@
 # Copyright 2015 AgileBG SAGL <http://www.agilebg.com>
 # Copyright 2015 innoviu Srl <http://www.innoviu.com>
 # Copyright 2018 Lorenzo Battistini
-# Copyright 2018 Sergio Zanchetta (Associazione PNLUG - Gruppo Odoo)
+# Copyright 2018 Sergio Zanchetta (Associazione PNLUG - Gruppo Odoo).
```

Comment:
?

Reply:
l'ho aggiunto perché volevo riavviare travis poiché non erano più disponibili i log, che andava in errore